openmetadata-ui/src/main/resources/ui/src/components/DataQuality/IncidentManager/IncidentManagerPageHeader/IncidentManagerPageHeader.component.tsx

```diff
@@ -76,6 +79,19 @@ const IncidentManagerPageHeader = ({
     initialAssignees,
   } = useActivityFeedProvider();
 
+  const columnName = useMemo(() => {
+    const isColumn = testCaseData?.entityLink.includes('::columns::');
+    if (isColumn) {
+      const name = replacePlus(
+        getColumnNameFromEntityLink(testCaseData?.entityLink ?? '')
```

Comment:
any reasons we are using replacePlus here?

Reply:
Removed `replacePlus` function call